lambdas/data-migration1/src/providers.ts

```diff
@@ -91,10 +91,10 @@ export const migrateProviderRecord = async (
     port: dynamoRecord.port,
     username,
     password,
-    globalConnectionLimit: dynamoRecord.globalConnectionLimit,
-    privateKey: dynamoRecord.privateKey,
-    cmKeyId: dynamoRecord.cmKeyId,
-    certificateUri: dynamoRecord.certificateUri,
+    global_connection_limit: dynamoRecord.globalConnectionLimit,
+    private_key: dynamoRecord.privateKey,
+    cm_key_id: dynamoRecord.cmKeyId,
```

Comment:
When you have a moment, review #1992 and see if there's any reason the approach there conflicts with this PR in your opinion.